src/Simplex/Messaging/Client.hs

```diff
@@ -69,7 +69,9 @@ data SMPClientConfig = SMPClientConfig
   { qSize :: Natural,
     defaultPort :: ServiceName,
     tcpTimeout :: Int,
-    smpPing :: Int
+    smpPing :: Int,
+    blockSize :: Int,
```

Comment:
why is `rsaKeySize` in agent config, but `blockSize` is in client's?

Reply:
because blockSize will also be used for transport block padding and rsaKeySize is only used for e2e encryption.
paddedMsgSize is calculated so that the whole command fits in this block, and message size is also consistent (to protect from observer that controls the server).
Server side RSA key is separate, it can be configured in the server.